internal/scorecard/interfaces.go

```diff
@@ -81,14 +81,6 @@ func (p externalPlugin) Run() scapiv1alpha1.ScorecardOutput {
 	return result
 }
 
-var basicTestsPlugin = basicOrOLMPlugin{
```

Comment:
@jmccormick2001 are any of these scorecard variables used for anything?

Reply:
I'd say they are safe to remove, I am not sure about the history of why they were there.